src/modules/index.js

```diff
@@ -1 +1,2 @@
-export { default as Auth } from "./Auth";
+export { default as Auth } from './Auth';
```

Comment:
I think we decided not to have this file.

Reply:
I think you said, don't use it for components, I'll remove this.